CONTRIBUTING.md

```diff
@@ -0,0 +1,325 @@
+# Contributing Guide
+
+## For Contributors
+
+Contributions are always welcome!
+If you have discovered a bug or have a feature request, [please create an issue on GitHub](https://github.com/kintone-labs/kintone-ui-component/issues/new/choose).  
+Pull requests are also welcome when you find trivial bugs or typos.
+
+We use English for all commit messages, code comments, issues, pull requests.
+
+### Setup
+This repository is a monorepo using npm.
+```sh
+% cd kintone-ui-component
+% npm ci
+```
+
+### Build
+The project should be Setup before Build.
+```sh
+% cd kintone-ui-component
+% npm run build:umd_prod
+```
+
+### Development
+The project should be Setup before Development.
+
+```sh
+% cd kintone-ui-component
+% npm run build:umd_dev
+```
+
+### Unit Test
+Unit test confirm DOM is correct and that if necessary throw error.  
+```sh
+% cd kintone-ui-component
+% npm run test
+```
+
+### Storybook
+Storybook Controls gives you a graphical UI to interact with a component's arguments dynamically, without needing to code. It creates an addon panel next to your component examples ("stories"), so you can edit them live.  
```

Comment:
How about writing like this and moving this section to the above of Build section? (order by actual dev step)
```suggestion
### Develop with Storybook
You can develop and check the behavior graphically with Storybook.
Storybook Controls gives you a graphical UI to interact with a component's arguments dynamically, without needing to code. It creates an addon panel next to your component examples ("stories"), so you can edit them live.
```